src/common/utility.c

```diff
@@ -447,7 +447,7 @@ static cairo_surface_t *_util_get_svg_img(gchar *logo, const float size)
     else
     {
       cairo_t *cr = cairo_create(surface);
-      cairo_scale(cr, factor, factor);
+      cairo_scale(cr, factor/ppd, factor/ppd);
       dt_render_svg(svg, cr, final_width, final_height, 0, 0);
```

Comment:
I think /ppd should go here instead:
`dt_render_svg(svg, cr, final_width / ppd, final_height / ppd, 0, 0);`
since most likely we need to use "dots" here, not real "pixels"

Reply:
makes sense, thanks a lot